openrtx/include/calibration/calibInfo_Mod17.h

```diff
@@ -32,11 +32,8 @@ typedef struct
 {
     uint16_t tx_wiper;
     uint16_t rx_wiper;
-    uint8_t  mic_gain;
-    uint8_t  tx_invert : 1,
-             rx_invert : 1,
-             _padding  : 6;
-}
-mod17Calib_t;
+    uint8_t mic_gain;
+    uint8_t tx_invert : 1, rx_invert : 1, _padding : 6;
```

Comment:
We should prevent the lint from doing this

Reply:
Right now this is happening because there are no comments after the field definition. The other cases where the newlines are maintained are there because of the comments. I'm unclear on whether we can force these to new lines without comments (or disabling clang-format for the block).

Will investigate feasibility here more later.